syntax = "proto3";

package fintekkers.models.position;

option java_multiple_files = true;

option java_outer_classname = "PositionStatusProtos";

enum PositionStatusProto {
  UNKNOWN=0;
  HYPOTHETICAL=1; //Hypothetical status means a transaction, tax lot or position that may never occur. This can be used to understand how potential actions could impact a portfolio
  INTENDED=2; //Intended status means a transaction, tax lot or position that is expected to occur if nothing changes. For example a fixed income bond that is expected to pay a coupon, or a security that is expected to mature in a specific point in the future
  EXECUTED=3; //Executed status means a transaction, tax lot or position that is the result of a legally binding transaction
}
